docs/guides/high-availability/concepts-active-passive-sync.adoc

```diff
@@ -0,0 +1,174 @@
+Monitoring the number of cache entries in both sites and the {project_name} log file can show when resynch would become necessary.
+
+Manual operations::
+* Manual operations that re-synchronize the {jdgserver_name} state between the sites will issue a full state transfer which will put a stress on the system (network, CPU, Java heap in {jdgserver_name} and {project_name}).
+
+== Questions and answers
+
+Why synchronous database replication?::
+A synchronously replicated database ensures that data written in the primary site is always available in the secondary site on failover and no data is lost.
+
+Why synchronous {jdgserver_name} replication?::
+A synchronously replicated {jdgserver_name} ensures that sessions created, updated and deleted in the primary site are always available in the secondary site on failover and no data is lost.
+
+Why is a low-latency network between sites needed?::
+Synchronous replication defers the response to the caller until the data is received at the secondary site.
+For synchronous database replication and synchronous {jdgserver_name} replication, a low latency is necessary as each request can have potentially multiple interactions between the sites when data is updated which would amplify the latency.
+
+Why active-passive?::
+Some databases support a single writer instance with a reader instance which is then promoted to be the new writer once the original writer fails.
+In such a setup, it is beneficial for the latency to have the writer instance in the same site as the currently active {project_name}.
+Synchronous {jdgserver_name} replication can lead to deadlocks when entries in both sites are modified concurrently.
+
+Is this setup limited to two sites?::
+This setup could be extended to multiple sites, and there are no fundamental changes necessary to have, for example, three sites.
+Once more sites are added, the overall latency between the sites increases, and the likeliness of network failures, and therefore short downtimes, increases as well.
+Therefore, such a deployment is expected to have worse performance and an inferior.
+For now, it has been tested and documented with blueprints only for two sites.
+
+Is a synchronous cluster less stable than an asynchronous cluster?::
+An asynchronous setup would handle network failures between the sites gracefully, while the synchronous setup would delay requests and will throw errors to the caller where the asynchronous setup would have deferred the writes to {jdgserver_name} or the database to the secondary site.
+However, as the secondary site would never be fully up-to-date with the primary site, this setup could lead to data loss during failover.
+This would include:
++
+--
+* Lost logouts, meaning sessions are logged in the secondary site although they are logged out in to the primary site at the point of failover when using an asynchronous {jdgserver_name} replication of sessions.
```

Comment:
Emails containing links to flows like password reset should also not work. Is it worth mentioning here too?